examples/python_nets/caffenet.py

```diff
@@ -0,0 +1,54 @@
+from caffe import layers as L, params as P, to_proto
```

Comment:
I was looking over this and from what I can tell the to_proto function from PR #1733 has moved
into NetSpec [here](https://github.com/BVLC/caffe/pull/2086/files#diff-821d83c04a94134a26dae418beeb622eR92). 

In this case NetSpec should be imported and all layer variables should be defined to it. This would then also print the layers in their correct order.

The alexnet function would look as follows:
def alexnet(lmdb, batch_size=256, include_acc=False):
    n = NetSpec()

    n.data, n.label = L.Data(source=lmdb, backend=P.Data.LMDB, batch_size=batch_size, ntop=2,
        transform_param=dict(crop_size=227, mean_value=[104, 117, 123], mirror=True))

    # the net itself
    n.conv1, n.relu1 = conv_relu(n.data, 11, 96, stride=4)
    n.pool1 = max_pool(n.relu1, 3, stride=2)
    n.norm1 = L.LRN(n.pool1, local_size=5, alpha=1e-4, beta=0.75)
    n.conv2, n.relu2 = conv_relu(n.norm1, 5, 256, pad=2, group=2)
    n.pool2 = max_pool(n.relu2, 3, stride=2)
    n.norm2 = L.LRN(n.pool2, local_size=5, alpha=1e-4, beta=0.75)
    n.conv3, n.relu3 = conv_relu(n.norm2, 3, 384, pad=1)
    n.conv4, n.relu4 = conv_relu(n.relu3, 3, 384, pad=1, group=2)
    n.conv5, n.relu5 = conv_relu(n.relu4, 3, 256, pad=1, group=2)
    n.pool5 = max_pool(n.relu5, 3, stride=2)
    n.fc6, n.relu6 = fc_relu(n.pool5, 4096)
    n.drop6 = L.Dropout(n.relu6, in_place=True)
    n.fc7, n.relu7 = fc_relu(n.drop6, 4096)
    n.drop7 = L.Dropout(n.relu7, in_place=True)
    n.fc8 = L.InnerProduct(n.drop7, num_output=1000)
    n.loss = L.SoftmaxWithLoss(n.fc8, n.label)

    if include_acc:
        n.acc = L.Accuracy(n.fc8, n.label)

    return n.to_proto()

BR, Max